pkg/controller/jobs/rayjob/rayjob_webhook_test.go

```diff
@@ -280,10 +290,28 @@ func TestValidateCreate(t *testing.T) {
 
 	for name, tc := range testcases {
 		t.Run(name, func(t *testing.T) {
+			ctx, _ := utiltesting.ContextWithLog(t)
+			builder := utiltesting.NewClientBuilder()
+			cli := builder.Build()
+			cqCache := cache.New(cli)
+			queueManager := queue.NewManager(cli, cqCache)
+			features.SetFeatureGateDuringTest(t, features.ManagedJobsNamespaceSelector, false)
+			features.SetFeatureGateDuringTest(t, features.LocalQueueDefaulting, tc.localQueueDefaulting)
+			if err := queueManager.AddLocalQueue(ctx, utiltesting.MakeLocalQueue("default", tc.job.Namespace).
+				ClusterQueue("cluster-queue").Obj()); err != nil {
+				t.Fatalf("failed to create default local queue: %s", err)
+			}
 			wh := &RayJobWebhook{
 				manageJobsWithoutQueueName: tc.manageAll,
+				queues:                     queueManager,
+				cache:                      cqCache,
+			}
+
+			job := tc.job.DeepCopy()
+			if err := wh.Default(t.Context(), job); err != nil {
+				t.Errorf("unexpected Default() error: %s", err)
```

Comment:
```suggestion
				t.Fatalf("unexpected Default() error: %s", err)
```
at this point it does not make sense to continue the test